### ISSUE TITLE
Sometimes uTox sound notification doesn't work until you restart uTox

### ISSUE THREAD

**Issue by towlie** [notsecure/uTox#750]
Using windows 7 x64
It worked a few times after getting called but somehow it happens that it doesn't work anymore after a few calls.
I don't know if it is easy to reproduce.

**Comment by GrayHatter** [comment]
@towlie can you make this happen again?

can I get a better description of what "doesn't work anymore" means for each?

**Comment by towlie** [comment]
@GrayHatter with sound I mean the ring tone when getting called. But it seems that it doesn't work in general on windows.
The second part is that if you kill explorer.exe and restart it then the tray icon of utox disappears.

**Comment by GrayHatter** [comment]
@towlie can you check your settings and make sure that ringtone is enabled?

**Comment by towlie** [comment]
Yes, on one laptop the audio notification was disabled somehow. So it's not a bug, but the tray icon bug persists. I will edit the title.

**Comment by towlie** [comment]
Whatever "unique environment" means I guess you can remove it because all windows users should have that behaviour.

**Comment by GrayHatter** [comment]
I've only ever seen explorer crash once or twice, and I never met anyone who's needed to restart explorer.exe without some sort of system crash/bug involved.

**Comment by towlie** [comment]
I sometimes have to because of a bug. Applications like qtox or skype are able to recreate the try icon.
